sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/commands.scala

```diff
@@ -79,7 +79,7 @@ case class AddJar(path: String) extends RunnableCommand {
     val hiveContext = sqlContext.asInstanceOf[HiveContext]
     hiveContext.runSqlHive(s"ADD JAR $path")
     hiveContext.sparkContext.addJar(path)
-    Seq.empty[Row]
+    Seq(Row(0))
```

Comment:
(I thought it may be better to comment at the original pr).

OK, I see. In future, let's make sure we also update the `output` if the result of a command is not an empty Seq (#5350 will change the schema for `AddJar`).